ansible-scylla-node/tasks/common.yml

```diff
@@ -299,6 +299,63 @@
       when: token_distributor is defined
   when: start_scylla_service is defined and start_scylla_service|bool
 
+- name: Check if authentication is enabled
+  lineinfile:
+    path: /etc/scylla/scylla.yaml
+    regexp: '^(authenticator: (.*PasswordAuthenticator.*|.*TransitionalAuthenticator.*))|(authorizer: (.*CassandraAuthorizer.*|.*TransitionalAuthorizer.*))'
```

Comment:
Huh?
Why do you have to process a YAML file as a plain text one?
Doesn't Ansible have YAML parsers for that?

Reply:
And it turns out it does: https://stackoverflow.com/questions/68217313/parse-yaml-files-in-ansible

Reply:
Fixed